integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -228,6 +228,51 @@ public static void checkServiceCreated(String serviceName, String domainNS, int
     }
   }
 
+  /**
+   * Resole a servive name inside a pod.
+   * @param serviceName service name to resolve
+   * @param podName name of the pod
+   * @param namespace namespace of the pod
+   * @throws Exception if a service name can not be resolved
+   */
+  public static void resolveServiceName(
+      String serviceName, String podName, String namespace, int numberOfRetries)
+      throws Exception {
+
+    String appLocationInPod = BaseTest.getAppLocationInPod();
+    String scriptLocationInPod = appLocationInPod + "/resolveServiceName.py";
+
+    // copy py script to check if managed service name can be resolved in admin pod
+    TestUtils.kubectlexec(podName, namespace," -- mkdir -p " + appLocationInPod);
```

Comment:
Surprisingly, this will break on k8s 1.16.  The `kubectl exec` command has always optionally taken a "--" as the separator between the kubectl arguments and the command that you want to run, but in 1.16 the parsing changes so that if you have this value it is taken as the beginning of the command.  We could fix this by always adding a "--" before the command on the `kubectl exec` command line.

Reply:
@rjeberhard currently the command looks like this 
```
kubectl -n usab-domainns-4 exec -it usab-domain-4-admin-server  -- mkdir -p /u01/oracle/apps
```
what will be the command based on your suggestion?